utils/data_loader.py

```diff
@@ -1,24 +1,46 @@
 from torch.utils.data import DataLoader
-from .heatmap_loader import HeatmapLoader
+from .heatmap_dataset import HeatmapDataset
 
-def CreateDataLoader(train_images, train_labels, val_images, val_labels, test_images, test_labels, raw_transform, transform, patch_shape, num_workers, batch_size, eps=0.00001, sigma=None, lower_bound=None, upper_bound=None):
```

Comment:
Function names should always be snake_case, i.e. `create_data_loader`, instead of CamelCalse, `CreateDataLoader`, in python.